src/main/java/org/fundacionjala/coding/dennis/FizzBuzz.java

```diff
@@ -0,0 +1,41 @@
+package org.fundacionjala.coding.dennis;
+
+/**
+ *
+ */
+public  final class FizzBuzz {
+
+    public static final int THREE = 3;
+    public static final int FIVE = 5;
+    public static final int MCM = THREE * FIVE;
+    public static final int LIMIT = 200;
+    /**
+     *
+     */
+    private FizzBuzz() {
+    }
+
+    /**
+     *
+     * @param limit limite
+     */
+
+    public static void fBuzz(int limit) {
+        StringBuilder lString;
+        for (int pt = 1; pt <= limit; pt++) {
+            lString = (pt % MCM == 0) ? (new StringBuilder("FizzBuzz"))
+                    : ((pt % THREE == 0) ? (new StringBuilder("Fizz"))
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Extract this nested ternary operation into an independent statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS3358)